interfaces/builtin/network_manager.go

```diff
@@ -81,6 +81,12 @@ network packet,
 
 # Needed to use resolvconf from core
 /sbin/resolvconf ixr,
+/run/resolvconf** rw,
```

Comment:
I'm not sure why this rule was in the ppp interface. It should be: `/run/resolvconf/{,**} rw,`. That said, is network-manager expected to create the /run/resolvconf directory? If not, use this instead:
```
/run/resolvconf/{,**} r,
/run/resolvconf/** w,
```